frontend/src/views/Main.vue

```diff
@@ -141,6 +160,66 @@ import { AbfrageDtoArtAbfrageEnum } from "@/api/api-client/isi-backend";
 export default class Main extends Vue {
   private speedDialOpen = false;
 
+  private iconOptions = {
+    shadowUrl: iconShadowUrl,
+    iconSize: [25, 41] as [number, number],
+    iconAnchor: [12, 41] as [number, number],
+    popupAnchor: [1, -34] as [number, number],
+    tooltipAnchor: [16, -28] as [number, number],
+    shadowSize: [41, 41] as [number, number],
+  };
+
+  private iconAbfrage = L.icon({ iconUrl: iconAbfrageUrl, ...this.iconOptions });
+
+  private iconBauvorhaben = L.icon({ iconUrl: iconBauvorhabenUrl, ...this.iconOptions });
+
+  private iconInfrastruktureinrichtung = L.icon({ iconUrl: iconInfrastruktureinrichtungUrl, ...this.iconOptions });
+
+  private geoJsonOptions: GeoJSONOptions = {
+    pointToLayer: (feature: EntityFeature, latlng) => {
+      let icon: L.Icon;
+      switch (feature.properties.type) {
+        case "ABFRAGE":
+          icon = this.iconAbfrage;
+          break;
+        case "BAUVORHABEN":
+          icon = this.iconBauvorhaben;
+          break;
+        case "INFRASTRUKTUREINRICHTUNG":
+          icon = this.iconInfrastruktureinrichtung;
+          break;
+        default:
+          return L.marker(latlng);
+      }
+      return L.marker(latlng, { icon });
+    },
+    onEachFeature: (feature: EntityFeature, layer) => {
+      layer.bindTooltip(feature.properties.name).openTooltip();
```

Comment:
Ich denke es macht sinn, dass im Tooltip die selben Inforamtionen wie in der Ergebnisliste stehen.

Reply:
Somit muss nicht beim Hovern kurz vor dem Klick mit einem Blick in die Ergebnisliste geprüft werden, ob dies wirklich die korrekte Entität ist.